firmware/controllers/sensors/sensor_type.h

```diff
@@ -71,6 +71,8 @@ enum class SensorType : unsigned char {
 
 	BarometricPressure,
 
+	ClutchUp,
```

Comment:
unrelated?

Reply:
Good catch.  Initially thought I needed to plumb it through before I saw that the digital inputs were getting set in engine.cpp.  This can/should be removed